webpack.demo.js

```diff
@@ -0,0 +1,54 @@
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    'angular2': [
+      'zone.js',
+      'reflect-metadata',
```

Comment:
> About putting extra script tags. I don't like that. 

This is not an argument :-) That is why we have webpack at the end of the day so we can bundle and minify the deps. 

> Unbundled for dev, bundled for prod. If we stick to script tags, we will have always dev ones or prod ones.

Not sure I understand this one...

Putting external dependencies in script tags has 2 advantages:
* enables effective caching in browsers
* avoid unnecessary work on WebPack side